tests/by-util/test_df.rs

```diff
@@ -107,6 +107,7 @@ fn test_type_option() {
         .args(&["-t", fs_type, "-t", "nonexisting"])
         .succeeds();
     new_ucmd!().args(&["-t", "nonexisting"]).fails();
+    new_ucmd!().args(&["-t", "nonexisting", "."]).fails();
```

Comment:
can you please update the test to check the actual error message?
would be something like
.stderr_contains("No file systems processed");

Reply:
Ok, fixed.